react-frontend/src/shared-components/peerly-logo/LogoComponent.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { Row } from "core-components/grid/GridComponent";
+import PlusSign from "shared-components/plus-sign/PlusSign";
+
+const Plus = styled.div`
+  & {
+    position: absolute;
+    width: 64px;
+    height: 77px;
+    top: 5%;
+    left: 5%;
+    transform: rotate(45deg);
+    font-size: 45px;
+    color: ${({ theme }) =>
+      (theme.text === "dark" && "var(--black)") ||
+      (theme.text === "light" && "var(--white)")};
+  }
+`;
+
+const Logo = styled.div`
```

Comment:
[Every component should follow “single responsibility principle”](https://github.com/joshsoftware/peerly/wiki/React-Development-Guidelines)